### DISCUSSION TITLE
Timeseries Data

### DISCUSSION THREAD

**Discussion by rwmwan** [supabase/supabase#6548] [Questions] [answered]
Hi Supabasers 

This might be a very greedy request but would love to see if there has been any thought of integrating with some form of timeseries database i do no there are some that use postgress

Would love to be able to generate our API's on supabase with the ability  to query time series data and mange retention policies this could help make working with timeseries data much easier for a lot of people and having the realtime functionality could add some very unique uses for supabase that we have not seen in other projects.

for me this would give the ability to merge our timeseries data and persistent data under one abstraction layer and make it easier to query both say for example sensor 6 for user 008 over 2 hours as a example :-)

**Comment by petehouston** [comment]
I'm also interested in this one :)

**Reply by vickkhera** [reply, marked as answer]
The `timescaledb` extension is one click away. I've not used it, but have read many good things about it.

**Reply by rwmwan** [reply]
This looks pretty good i will have to give it a test thanks very much @vickkhera 

Just for ref incase anyone else wants to have a look link to timescale extension docs below 

https://supabase.com/docs/guides/database/extensions/timescaledb

**Reply by madebyfabian** [reply]
Now that timescaledb is deprecated in Postgres 17 instances (see https://supabase.com/docs/guides/database/extensions/timescaledb) what would be the best way to store timeseries data now?
